save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestRepository.kt

```diff
@@ -2,8 +2,17 @@ package org.cqfn.save.backend.repository
 
 import org.cqfn.save.entities.Test
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 
 /**
  * Repository of tests
  */
-interface TestRepository : JpaRepository<Test, String>
+interface TestRepository : JpaRepository<Test, String> {
+    /**
+     * Method to retrieve ready batches
+     *
+     * @return List of Tests
+     */
+    @Query(value = "select * from test inner join test_execution on test.id = test_execution.test_id and test_execution.status = 'READY'", nativeQuery = true)
```

Comment:
Batch should also be limited by a fixed size (let's say 20 ids for now), and TestService should store offset for the last test